docs/preset-env.md

```diff
@@ -392,7 +392,9 @@ Don't add polyfills automatically per file, and don't transform `import "core-js
 
 Added in: `v7.4.0`
 
-`2`, `3` or `{ version: 2 | 3, proposals: boolean }`, defaults to `2`.
+`2`, `3`, `3.minor`, or `{ version: 2 | 3, proposals: boolean }`, defaults to `2`.
```

Comment:
This probably isn't the best way to document it, but I couldn't figure out a better way

Reply:
Maybe we can just put "`string` or `{ version: string, proposals: boolean }` here, and then list the possible versions more in prose?

Reply:
We should recommend string, e.g. `"3.10"`, for minor version usage here, otherwise we can't differentiate 3.1 and 3.10.

Reply:
I'll update it to state `string` or `{ version: string, proposals: boolean }` and then update the note to expand upon that